### PR TITLE
Add basic support for plain text response format

### PR DESCRIPTION
**Pull request by sakjur**
Not particularly refined, but it doesn't crash®

### CODE REVIEW

File: swagger2markdown.py [hunk 1]
```diff
@@ -276,10 +276,14 @@ def convert(self):
                             responseRef = responses['200']['schema']['$ref']
                             response = findDefintion(responseRef, self.definitions)
                             responseType = 'object'
-                        elif '$ref' in responses['200']['schema']['items']:
+                        elif 'items' in responses['200']['schema'] and '$ref' in responses['200']['schema']['items']:
                             responseRef = responses['200']['schema']['items']['$ref']
                             response = findDefintion(responseRef, self.definitions)
                             responseType = responses['200']['schema']['type']
+                        elif 'type' in responses['200']['schema']:
```
Comment (jelgh) [inline swagger2markdown.py]: `and type=='string'` right?

**Comment by sakjur** [comment]
Yeah, that would make sense :)

**Comment by sakjur** [comment]
@jelgh Updated :)

File: swagger2markdown.py (hunk 1 above)
Comment (sakjur) [reply swagger2markdown.py]: (I fixed this, but in https://github.com/tink-ab/tink-public-documentation/pull/6/commits/4a804b6757473b87e54e244793f796954c82c27a#diff-2212cc6cf19d06602f12dbbb3688a91fR353 )

**Pull request merged**